Generator_pgn.py

```diff
@@ -0,0 +1,82 @@
+import chess
+import torch.multiprocessing as mp
+import bz2
+
+class Generator_pgn():
```

Comment:
Without good arguments, standard Python naming conventions should be used: https://www.python.org/dev/peps/pep-0008/#class-names